Sixteen750/src/main/java/org/firstinspires/ftc/sixteen750/opmodes/auto/AutoTest.java

```diff
@@ -39,7 +39,8 @@ public void uponInit() {
     }
 
     public void uponStart() {
-        robot.prepForStart();
+        robot.horizontalSlidesSubsystem.slidesRetract();
```

Comment:
What's the win of doing just this, vs. running the full prepForStart set of stuff? Either way, I'd prefer you did a "fastPrep" so that the code could be more easily tweaked & shared between auto opmodes...